zuul.d/jobs.yaml

```diff
@@ -2191,3 +2191,20 @@
       - OS:ubuntu-xenial
       - Arch:arm64
       - BuildType:Compile
+
+# Spark arm64 build and run unit tests periodic job
+- job:
+    name: spark-unit-test-arm64
```

Comment:
`build-and-test` ?

Reply:
According https://docs.openlabtesting.org/contributors/development-guide#job-naming-notations * The test type is the type of test to run, e.g. acceptance test, integration test, unit test or build
So I will remain the name of 'spark-unit-test-arm64'

Reply:
this job is not only to run test, and also to check `build`, right ?

Reply:
I think the current job name is OK, just like `cloud-provider-openstack-acceptance-test-e2e-conformance`, it compile, then run acceptance test.

Reply:
OK, +1